src/components/ProductsSidebar.js

```diff
@@ -0,0 +1,42 @@
+import productCategoriesData from '../mocks/en-us/product-categories.json';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimesCircle } from '@fortawesome/free-solid-svg-icons'
+
+const ProductsSidebar = (props) => {
```

Comment:
I think just one comment here, remember `divide and conquer` 
What in my opinion changes the most from functional components to class components is that FC tend to divide as much as is possible a component into multiple smaller components ie a categories or chips component